catroid/src/main/java/org/catrobat/catroid/content/actions/SetBackgroundLookAction.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2018 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.catroid.content.actions;
+
+import org.catrobat.catroid.ProjectManager;
+import org.catrobat.catroid.common.LookData;
+import org.catrobat.catroid.content.EventWrapper;
+import org.catrobat.catroid.content.Sprite;
+import org.catrobat.catroid.content.eventids.SetBackgroundEventId;
+
+public class SetBackgroundLookAction extends EventAction{
```

Comment:
i have no idea why lint isnt complaining, but "EventAction{" should have a space inbetween EventAction and the bracket

Reply:
Me neither!
However, looking through the documentation, there is no rule or setting to force this :flushed:

…actually, there is one. By adding left and right curly braces to [WhitespaceAround](http://checkstyle.sourceforge.net/config_whitespace.html#WhitespaceAround). This will also result in a violation for e.g.
```java
public @interface RotationStyle {}
```

which however is already a violation according to the Android Studio formatter. I like this style and tried to adapt the AS formatter a long time ago without success. The changes always effected other code formats as well…

So, btt. I'll do some additional research here. Maybe write a GitHub issue.